### PR TITLE
first commit search bar

### PR DESCRIPTION
**Pull request by panianchuk**
https://panianchuk.github.io/layout_search-bar-airbnb/
https://panianchuk.github.io/layout_search-bar-airbnb/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,54 @@
...
+        action="#"
+        method="post"
+        data-qa="big"
+      >
+        <label for="search-input-big">
+          <img 
+          src="images/Search.svg" 
+          alt="Search" 
+          class="big-icon"
+          >
```
Comment (SanyaBratashchuk) [inline src/index.html:26]:
```suggestion
          <img 
            src="images/Search.svg" 
            alt="Search" 
            class="big-icon"
          >
```

Comment (SanyaBratashchuk) [reply src/index.html:26]: Fix it everywhere

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+  
+* {
+  box-sizing: border-box;
+  font-family: Avenir, Helvetica, sans-serif;
+  font-style: normal;
```
Comment (SanyaBratashchuk) [inline src/style.css:16]: move it to the `html` tag selector

File: src/index.html [hunk 3]
```diff
@@ -11,6 +11,54 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    
+    <div class="search-form-big">
```
Comment (SanyaBratashchuk) [inline src/index.html:15]: why do you need this wrapper you can just apply styles directly to the form element

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]
image

**Comment by panianchuk** [comment]
Changed 
https://panianchuk.github.io/layout_search-bar-airbnb/

**Review by SanyaBratashchuk — APPROVED** [review]

**Pull request closed without merging**